src/citrine/informatics/catalyst/insights.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+
+from citrine._serialization import properties
+from citrine._serialization.serializable import Serializable
+from citrine.informatics.catalyst.language_model import LanguageModelChoice
+
+
+class InsightsRequest(Serializable["InsightsRequest"]):
+    """A query to the model insights."""
+
+    question = properties.String("question")
+    temperature = properties.Optional(properties.Float, "temperature", default=0.0)
+    language_model = properties.Optional(
+        properties.Enumeration(LanguageModelChoice),
+        "language_model",
+        default=LanguageModelChoice.GPT_35_TURBO,
+    )
+    n_documents = properties.Optional(properties.Integer, "n_documents", default=5)
+    response_size = properties.Optional(
+        properties.Integer, "response_size", default=100
+    )
+
+    def __init__(
+        self,
+        *,
+        question: str,
+        temperature: Optional[float] = 0.0,
+        language_model: Optional[
+            LanguageModelChoice
+        ] = LanguageModelChoice.GPT_35_TURBO,
```

Comment:
This formatting is super gross and confusing. It's stuff like this why I'm skeptical of tools like `black`...